docs/sources/outgoing-webhooks/_index.md

```diff
@@ -365,7 +368,7 @@ It will result in the following (Invalid JSON due to single quotes):
 
 ```json
 {
-  "labels": {'region': 'eu-1', 'alertname': 'TestAlert'}
```

Comment:
This is supposed to have invalid json as an example of using tojson() on a dict to fix it.